src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts

```diff
@@ -454,4 +465,29 @@ export class ChallengeleaderboardComponent implements OnInit, AfterViewInit {
       () => {}
     );
   }
+
+   // Update leaderboard decimal precision value
+   updateLeaderboardDecimalPrecision(event: MatSliderChange) {
+    const API_PATH = this.endpointsService.particularChallengePhaseSplitUrl(this.selectedPhaseSplit['id']);
+    const SELF = this;
+    SELF.leaderboardPrecisionValue = event.value;
+    const BODY = JSON.stringify({
+      'leaderboard_decimal_precision': SELF.leaderboardPrecisionValue
+    });
+    SELF.apiService.patchUrl(
+      API_PATH,
+      BODY
+    ).subscribe(
+        data => {
+          SELF.refreshLeaderboard();
```

Comment:
I think we don't need to refresh leaderboard cause it's just update the leaderboard precision value and the effect will be shown on the score metric as we are updating `leaderboardPrecisionValue`